core-services/various_rbac/admin_user_roles.yaml

```diff
@@ -123,6 +123,20 @@ items:
   - jlebon
   - peterbaouoft
   - smarterclayton
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: rhcos-admins
+  users:
+  - ashcrow
+  - bh7cw
+  - cgwalters
+  - dustymabe
+  - darkmuggle
+  - jlebon
+  - miabbott
+  - mike-nguyen
+  - rravanelli
```

Comment:
Double check this account. It 404's for me.

Reply:
Gawk, done. I used the wrong username. https://github.com/ravanelli checks out now.